LICENSE

```diff
@@ -199,17 +199,3 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-
```

Comment:
For source release, we now bundled

src/main/resources/org/apache/kyuubi/ui/swagger/*
src/main/resources/org/apache/kyuubi/ui/static/assets/fonts/*
src/main/resources/org/apache/kyuubi/ui/static/semantic.min.css
src/main/resources/org/apache/kyuubi/ui/static/semantic.min.js
src/main/resources/org/apache/kyuubi/ui/static/icon.min.css
src/main/resources/org/apache/kyuubi/ui/static/jquery-3.6.0.min.js

Reply:
This is not a part of 1.5

Reply:
Let's fix the master first, then subtracting for 1.5

Reply:
where are `src/main/resources/org/apache/kyuubi/ui/static/assets/fonts/*` comes from? and what's the license of them?

Reply:
see icon.min.css's header